dockers/docker-database/base_image_files/monit_database

```diff
@@ -3,5 +3,5 @@
 ## process list:
 ##  redis_server
 ###############################################################################
-check process redis_server matching "/usr/bin/redis-server"
-    if does not exist for 5 times within 5 cycles then alert
+check program container_process_redis_server with path "/usr/bin/process_checker database redis-server /usr/bin/redis-server"
```

Comment:
Does Monit accept colons in the program name? If so, I suggest changing the names for readability to the format, `<container>:<program>`. E.g., this one would be `database:redis-server`

Edit: I found that Monit does NOT accept colons. Instead, use `|` as the separator. E.g., `database|redis-server`

Reply:
Great suggestion! Will update.